go.mod

```diff
@@ -35,4 +35,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220127135604-0fab356f7a15
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20220127135755-5e21549c9cb8
```

Comment:
Remove these

Reply:
done in https://github.com/codeready-toolchain/toolchain-e2e/pull/462/commits/a02523fa1c5bfc90a09c2b3a50ce9d59dfd43692